### DISCUSSION TITLE
[BUG]快照备份失败变20mb

### DISCUSSION THREAD

**Discussion by Zevo77** [1Panel-dev/1Panel#6479] [Q&A]
### 联系方式

V：Qianzw7

### 1Panel 版本

v1.10.13-lts

### 问题描述

1panel快照用webdav上传alist后变成20mb
![IMG_8757](https://github.com/user-attachments/assets/6a4a8b32-d23b-41c9-959e-211172430fa2)

### 重现步骤

1

**Comment by wanghe-fit2cloud** [comment]
1、查看一下 `日志审计` - `系统日志`，看是否有 Error 相关信息。
2、再详细描述一下具体操作和使用场景。
